frontend/frontend/src/views/CompetingSpecies/CompetingSpecies.vue

```diff
@@ -426,8 +435,10 @@ export default {
       this.getAllPresets()
     }
     //Set initial values, calling initialConditions computed property to be inherited by charts
-    this.N1_0 = this.simParamData[0]
-    this.N2_0 = this.simParamData[4]
+    const defaultN1_0 = this.simParamData[0]
+    const defaultN2_0 = this.simParamData[4]
+    this.N1_0 = defaultN1_0
+    this.N2_0 = defaultN2_0
```

Comment:
```suggestion
    const defaultN1_0_index = 0
    const defaultN2_0_index = 4
    this.N1_0 = this.simParamData[defaultN1_0_index]
    this.N2_0 = simParamData[defaultN2_0_index]
```

You still have those magic numbers of 0 and 4 with your change, this was what I was meaning

Reply:
Not crucial though